packages/frontend/web/browser.ts

```diff
@@ -41,27 +40,65 @@ const go = () => {
         sendGaPageView();
     };
 
-    isAdBlockInUse()
-        .then((adBlockInUse: boolean) => {
-            if (adBlockInUse) {
-                hydrate();
-            } else {
-                getRaven().context(
-                    {
-                        tags: {
-                            feature: 'dotcom-rendering',
-                        },
-                    },
-                    hydrate,
-                );
+    getRaven()
+        .catch(err => {
+            hydrate();
```

Comment:
Just thought... if we `catch` here, then `hydrate()`, we'll drop into the `then` callback and `hydrate()` again. Perhaps we should just `console.log(err)` or do nothing at this point?